thirdparty/README.md

```diff
@@ -173,6 +173,22 @@ comments. Apply the patches in the `patches/` folder when syncing on newer upstr
 commits.
 
 
+## glad
+
+- Upstream: https://github.com/Dav1dde/glad
+- Version: 2.0.2 (f237a2bfcec0d9b82b90ec9af4af265c40de7183, 2022)
+- Generator License: MIT, Apache, Khronos
+- Generated files license: PD/WTFPL/CC0, Apache
```

Comment:
We only include generated files so we don't need to mention the generator license.

For the public domain part, I would pick one since we're given the choice - less stuff to document in `COPYRIGHT.txt`, especially as we don't use the WTFPL anywhere yet.
```suggestion
- License: CC0 1.0 and Apache 2.0
```

Reply:
And you should also add an entry in `COPYRIGHT.txt`. I believe this would be correct:
```diff
diff --git a/COPYRIGHT.txt b/COPYRIGHT.txt
index 3f4981b7fb..2c9e55ca23 100644
--- a/COPYRIGHT.txt
+++ b/COPYRIGHT.txt
@@ -202,6 +202,12 @@ Comment: The FreeType Project
 Copyright: 1996-2022, David Turner, Robert Wilhelm, and Werner Lemberg.
 License: FTL
 
+Files: ./thirdparty/glad/
+Comment: glad
+Copyright: 2013-2022, David Herberth
+ 2013-2020, The Khronos Group Inc.
+License: CCO-1.0 and Apache-2.0
+
 Files: ./thirdparty/glslang/
 Comment: glslang
 Copyright: 2015-2020, Google, Inc.
```